docs/source/apis/saas_api.rst

```diff
@@ -0,0 +1,137 @@
+SaaS API Deployment Guide
+=========================
+
+This documentation serves as a guide for Unstructured SaaS API users to register and manage API Key, as well as the billing information. It contains all the necessary instructions and resources to facilitate a smooth setup and deployment process of Hosted API on Unstructured infrastructure.
+
+
+New User Sign-up
+----------------
+
+1. Navigate to the `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ page on our website.
+2. Fill out the registration form with your details:
+
+   - **First Name**: Enter your given name.
+   - **Last Name**: Enter your family or surname.
+   - **Email**: Provide a valid email address where we can send you confirmations and communications.
+   - **Company**: Enter the name of your company or organization.
+
+3. Optional Preferences:
+
+   - [] Check the box to sign up for future marketing emails and product updates.
+   - [] Check the box to sign up for weekly usage usage emails.
+
+4. Terms and Conditions:
+
+   - [] Read and accept our terms and conditions by checking the appropriate box to proceed.
+
+5. Once all information is entered and preferences and terms are accepted, click the "Sign Up" button to create your account.
```

Comment:
they should have done all of the proceeding steps so no need to reiterate here, right?

```suggestion
5. Click the "Sign Up" button to create your account.
```